src/frontend/src/lib/components/pages/SettingsSignedIn.svelte

```diff
@@ -97,21 +109,20 @@
 	</KeyValuePairInfo>
 </div>
 
-{#if POUH_ENABLED}
-	<div class="mt-8">
+{#if POUH_ENABLED && nonNullish($userProfileStore)}
```

Comment:
should we created a derived store instead of `nonNullish($userProfileStore)`?

Reply:
Maybe, but let's discuss after the refactor of the user profile store on Monday.